examples/plugins/notifications_slack/notifications.py

```diff
@@ -0,0 +1,279 @@
+from __future__ import print_function
+
+import logging
+import os
+from datetime import date, datetime, timedelta
+from typing import List, Optional
+
+import pluggy
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+from slack_sdk.signature import SignatureVerifier
+
+from api.models import AccessRequest, OktaGroup, OktaUser, RoleGroup
+
+notification_hook_impl = pluggy.HookimplMarker("access_notifications")
+logger = logging.getLogger(__name__)
+
+# Initialize Slack client and signature verifier
+slack_token = os.environ["SLACK_BOT_TOKEN"]
+signing_secret = os.environ["SLACK_SIGNING_SECRET"]
```

Comment:
Where do you use this? I think this is only needed if slack send you back messages

Reply:
i removed this line and line 22; deployment worked fine